core/src/main/java/org/infinispan/configuration/global/GlobalJmxStatisticsConfiguration.java

```diff
@@ -64,6 +64,10 @@ public TypedProperties properties() {
       return properties.get();
    }
 
+   /**
+    * @deprecated Since 10.1, please set a unique {@link GlobalConfiguration#cacheManagerName()} instead.
```

Comment:
It's strange not to mention setting a unique domain first :) A unique cache manager name helps also.

Reply:
Done